src/i18n/en-US.json

```diff
@@ -1175,6 +1175,9 @@
   "tooltipConversationFile": "Add file",
   "tooltipConversationInfo": "Conversation info",
   "tooltipConversationInputPlaceholder": "Type a message",
+  "tooltipConversationInputOneUserTyping": "{{user1}} is typing...",
+  "tooltipConversationInputTwoUserTyping": "{{user1}} and {{user2}} are typing...",
+  "tooltipConversationInputMoreThanTwoUserTyping": "{{user1}} and 2 more are typing...",
```

Comment:
We can maybe put `activeUsers.lenght`into a variable and use that instead of '2'?